src/app/pages/TopicPage/TopicPage.jsx

```diff
@@ -99,43 +93,21 @@ const TopicPage = ({ pageData }) => {
               title: curationTitle,
               link,
               position,
-              visualStyle = NONE,
+              visualStyle,
             }) => {
-              const component = getComponent(visualStyle, visualProminence);
-
               return (
                 <React.Fragment key={`${curationId}-${position}`}>
-                  {(() => {
-                    switch (component) {
-                      case MESSAGE_BANNER:
-                        return (
-                          <MessageBanner
-                            heading={curationTitle}
-                            position={position}
-                            description={summaries[0].description}
-                            link={summaries[0].link}
-                            linkText={summaries[0].title}
-                            image={summaries[0].imageUrl}
-                          />
-                        );
-                      case NOT_SUPPORTED:
-                        return null;
-                      default:
-                        return (
-                          <Curation
-                            headingLevel={curationTitle && 3}
-                            visualStyle={visualStyle}
-                            visualProminence={visualProminence}
-                            promos={summaries}
-                            title={curationTitle}
-                            topStoriesTitle={topStoriesTitle}
-                            position={position}
-                            link={link}
-                            curationLength={curations && curations.length}
-                          />
-                        );
-                    }
-                  })()}
+                  <Curation
```

Comment:
Quick question - why has this switch case which optionally returned the MessageBanner been removed?

Reply:
👋 Isabella, great question! All of that functionality has been moved into the Curation component. Previously the Curation component only catered for the hierarchical & simple curation grid, but then we realised that anything that can be configured in Tipo should be rendered by the Curation component, which is why we moved it. 

We're planning to reuse the Curation component for the "new" tipo-powered homepage, and we didn't want to have to duplicate the message banner functionality unless necessary.

Reply:
Ah that makes sense! Thanks :)